index.html

```diff
@@ -51,7 +51,206 @@ <h1 class="first">I'm Recillah Glad To See You!</h1>
+                            <img class="counter" src="Images/Counter.png" alt="dot\fullstop">
+                            <h4 class="year-label">2015</h4>
+                        </div>
+                        <p class="primary-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <ul class="tags">
+                            <li class="tag">html</li>
+                            <li class="tag">css</li>
+                            <li class="tag">javascript</li>
+                        </ul>
+                        <button class="action">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="single-card">
+                <div class="card-works">
+                    <a href="#">
+                        <img class="cards-pic" src="Images/Snapshoot-Portfolio-1.png" alt="Snapshoot Portfolio 1">
+                    </a>
+                </div>
+                <div class="left-block">
+                    <div class="primary-text">
+                        <div>
+                            <h2 class="project-title">Tonic</h2>
+                        </div>
+                        <div class="frame-2">
+                            <a class="client" href="#"> CANOPY</a>
+                            <img class="counter" src="Images/Counter.png" alt="dot\fulstop">
+                            <h3 class="role">Back End Dev</h3>
+                            <img class="counter" src="Images/Counter.png" alt="dot\fullstop">
+                            <h4 class="year-label">2015</h4>
+                        </div>
+                        <p class="primary-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <ul class="tags">
+                            <li class="tag">html</li>
+                            <li class="tag">css</li>
+                            <li class="tag">javascript</li>
+                        </ul>
+                        <button class="action" id="0">See Project</button>
+                    </div>
+                </div>
+            </div> 
+        </section>
+        <div class="big-container">
+            <div class="container"></div>
+        </div>
+        <section class="about-myself">
+            <div class="top">
+                <h2 class="primary-text-1">About Myself</h2>
+                <p class="secondary-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p> 
+                <div class="social-media-1">
+                    <a href="#" class="supporting-text">LET'S CONNECT</a>
```

Comment:
- [ ] I think it would be better to use a header tag hx (`<h2>`, `<h3>`, ...) for this element (` <a href="#" class="supporting-text">LET'S CONNECT</a>`) instead of `<a>` tag 👌. because It's considered a subtitle.